api/apps/geoprocessing/src/export/pieces-exporters/project-metadata.ts

```diff
@@ -1,19 +1,67 @@
 import { Injectable } from '@nestjs/common';
+import { InjectEntityManager } from '@nestjs/typeorm';
+import { EntityManager } from 'typeorm';
+import { Readable } from 'stream';
+
 import { ClonePiece, JobInput, JobOutput } from '@marxan/cloning';
+import { FileRepository } from '@marxan/files-repository';
+
+import { geoprocessingConnections } from '@marxan-geoprocessing/ormconfig';
+
 import { PieceExportProvider, PieceProcessor } from '../pieces/piece-processor';
+import { isLeft } from 'fp-ts/Either';
 
 @Injectable()
 @PieceExportProvider()
 export class ProjectMetadata extends PieceProcessor {
+  constructor(
+    private readonly fileRepository: FileRepository,
+    @InjectEntityManager(geoprocessingConnections.apiDB)
+    private readonly entityManager: EntityManager,
+  ) {
+    super();
+  }
+
   isSupported(piece: ClonePiece): boolean {
     return piece === ClonePiece.ProjectMetadata;
   }
 
   async run(input: JobInput): Promise<JobOutput> {
-    console.log(`exporting project metadata`, input);
+    const projectData: Array<{
+      name: string;
+      description: string;
+    }> = await this.entityManager.query(
```

Comment:
why not a plain `find()`?

Reply:
@hotzevzl glad you asked. We are in `geoprocessing` and `project` entity is in API ... due to references all around, we would need to move all the entities into `libs` 🤷

Reply:
yes, of course 😬 😅